src/Command/CreateTestCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace TrendingRepos\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CreateTestCommand extends Command {
```

Comment:
Too much duplication with `CreateControllerCommand` it's almost the same, maybe both should extend the same class ? will think about this later after testing part .